.circleci/config.yml

```diff
@@ -84,7 +84,7 @@ jobs:
             - 1-dependencies-{{ checksum "package.json" }}
       - restore_cache:
           keys:
-            - 5-dependencies-example-{{ checksum "example/package.json" }}
+            - 7-dependencies-example-{{ checksum "example/package.json" }}
```

Comment:
this is required so that the build machine will force the update to the latest native plugin code rather than using anything in cache, since the version number did not change in the package.json but the contents of our plugin native code did change (6 was used in a test build, so moving to 7 here)

side note: Im increasingly inclined to take out the example app node_module cache entirely, as it may save a few seconds on the build, but it can lead to confusion and incorrect output in the example app builds that takes longer to comprehend

Reply:
I would agree to take it all out, its not like we are building this constantly